src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(AllColors.values().length);
+        return String.valueOf(AllColors.values()[index]);
```

Comment:
it’s better to use standard method of enum name() that will be returning always String representation of concrete enum constant.